.license

```diff
@@ -0,0 +1,4 @@
+/**
```

Comment:
Replace the content of the file, use `vendor/spryker/decimal-object/.license` as a source.